src/Pages/PrivacyPolicy.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import styled from 'styled-components';
+import BackBtn from "../components/BackBtn/BackBtn";
+
+
+const PrivacyPolicy = () => {
+  return (
+   <>
+    <div className="py-8 mb-5 w-[50vw] justify-start ml-auto mr-auto mt-10">
+    <div>
+      <BackBtn Page={"Privacy Policy"} />
+      <PolicyWrapper>
+      
+      <section>
+      <p>Last updated: May 17, 2024</p>
+        <p>Welcome to NactoCare. We are committed to protecting your personal information and your right to privacy. If you have any questions or concerns about our policy or our practices with regards to your personal information, please contact us at nactoreorganization@gmail.com</p>
+      </section>
+      <section>
+        <h2>Information We Collect</h2>
+        <p>We collect personal information that you provide to us, such as name, address, contact information, passwords and security data, and payment information. We collect personal information when you register for our services, express an interest in obtaining information about us or our products and services, participate in activities on the Services, or otherwise contact us.</p>
```

Comment:
Looks good except that we don't collect passwords or security data. Cryptographic hashes will be used for user verification so that we don't need to collect passwords or even the salts.